EposeaLocalBackend/appsettings.json

```diff
@@ -6,6 +6,9 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "ConnectionStrings": {
+    "BloggingDatabase": "User ID=postgres;Password=root;Host=localhost;Port=5432;Database=eposea;"
```

Comment:
rename to LocalDatabase, also you should use `local_database` instead of `eposea` db